### ISSUE TITLE
ios 11 support?

### ISSUE THREAD

**Issue by JimCook57** [cubiq/add-to-homescreen#268]
Is anyone working on an update to support IOS 11?

**Comment by briansa** [comment]
I don't think so, but if you look at the last commit you'll see what needs to be done and you can always make a pull request.

https://github.com/cubiq/add-to-homescreen/commit/aa54e4683e58ca20c78ccfd2eee8718416dad882

**Comment by miclaus** [comment]
+1

**Comment by holtkamp** [comment]
https://github.com/cubiq/add-to-homescreen/pull/270

**Comment by amitailanciano** [comment]
Kinda crappy hack, but this is what I ended up doing for now

```
...
onShow: () => {
    $('.ath-container.ath-ios').addClass('ath-ios8');
}
...
```
